test/library/packages/Sort/RadixSort/radixsortMSB.chpl

```diff
@@ -0,0 +1,29 @@
+ use BitOps;
+ use Sort;
+ use Random;
+ use radixMSBModule;
+ use Time;
+ config const printStats = true;
+
+ proc main() {
+  
+   var array:[1..10000] int; 
```

Comment:
It might be nice to make the array size a `config const` so that it's easy to test out different sizes. We may even integrate this into our performance test eventually, with multiple tests timing different problem sizes.